### ISSUE TITLE
Variable arg list for llk_math_eltwise_unary_sfpu_*_param

### ISSUE THREAD

**Issue by rtawfik01** [tenstorrent/tt-metal#9036]
For this function: `llk_math_eltwise_unary_sfpu_*_param`, a new one is made for every variable number of params:
```
tt_metal/hw/ckernels/<arch>/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_0_param.h
tt_metal/hw/ckernels/<arch>/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_1_param.h
tt_metal/hw/ckernels/<arch>/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_2_param.h
tt_metal/hw/ckernels/<arch>/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_3_param.h
tt_metal/hw/ckernels/<arch>/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_5_param.h
```

and the files contain identical code, except for the call of:
```
            func(param_n.....);
```

All the above files and functions should be consolidated into one function:
```
inline void llk_math_eltwise_unary_sfpu_params
```
That should be able to take a variable arg list for the function parameters. This way we can remove all duplicated code, and remove need of external users to add another file to take an N number of params.

@ttmtrajkovic @rdjogoTT

**Comment by rtawfik01** [comment]
Thanks @yan-zaretskiy for demonstrating how variable arg lists can reduce duplication for these functions! First PR for wormhole b0 is here: https://github.com/tenstorrent/tt-metal/pull/9065

**Comment by rtawfik01** [comment]
Merged #9065 & #9078
